packages/amplify-codegen/src/utils/updateAmplifyMeta.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
Can we use a helper function available in  cli core to update the amplify meta file